WhaleSpotting/Services/SightingsService.cs

```diff
@@ -75,11 +76,28 @@ public async Task<SightingResponseModel> ConfirmSighting(int id)
             {
                 return null;
             }
-            
+
             sighting.Confirmed = true;
             _context.SaveChanges();
 
             return new SightingResponseModel(sighting);
         }
+
+        public async Task<IEnumerable<string>> GetSpeciesByCoordinates(string latitude, string longitude)
+        {
+            var fiftyKmInCoords = 0.45;
+            var upperLatitude = double.Parse(latitude) + fiftyKmInCoords;
+            var lowerLatitude = double.Parse(latitude) - fiftyKmInCoords;
+
+            var upperLongitude = double.Parse(longitude) + fiftyKmInCoords;
+            var lowerLongitude = double.Parse(longitude) - fiftyKmInCoords;
+
+            var sightings = await _context.Sightings
+                .Where(s => s.Latitude > lowerLatitude && s.Latitude < upperLatitude)
+                .Where(s => s.Longitude > lowerLongitude && s.Longitude < upperLongitude)
+                .ToListAsync();
+
+            return sightings.Select(sightings => sightings.Species.ToString()).Distinct();
```

Comment:
As in the endpoint, can we remove the ToString here and return the enum

Reply:
Addressed.